packages/twilio-run/src/config/global.ts

```diff
@@ -68,12 +69,20 @@ export function readSpecializedConfig<T extends CommandConfigurationNames>(
     opts &&
     opts.username &&
     projectsConfig &&
-    projectsConfig[opts.username]
+    (projectsConfig[opts.username] ||
+      projectsConfig[`${opts.username}:${opts.region}`])
   ) {
-    result = {
-      ...result,
-      ...projectsConfig[opts.username],
-    };
+    if (projectsConfig[`${opts.username}:${opts.region}`]) {
+      result = {
+        ...result,
+        ...projectsConfig[`${opts.username}:${opts.region}`],
+      };
+    } else if (opts.region === 'us1' || opts.region === undefined) {
```

Comment:
do we need to put this first? not sure if we're allowing `username:us1`, but it seems like the `opts.region === 'us1'` case would fall into the if block, so we should remove it or swap the if and else if blocks.

Reply:
I think logic you mentioned would fail in following case:
`region=us1` and the config file with `account:us1`.

And yes, we will be allowing accountsid:us1